### ISSUE TITLE
Bot Strength increase option (cheat?)

### ISSUE THREAD

**Issue by megastruktur** [adamqqqplay/dota2ai#104]
Is it possible bots could get **more gold/XP via some option**? I'd like bots to cheat as it's pretty easy to play versus them at this point.
Yesterday I saw the situation when Bane solo laning while his teammates were fighting, got his lvl8 when all the others were 11+

My personal list of bots strength is below.

**The list of pleasantly strong:**
- Bounty Hunter
- Bristleback ( <3 )
- Jakiro
- Phantom Assassin ( <3 )
- Phantom Lancer
- Pugna
- Shadow Demon
- Shadow Fiend
- Undying
- Venomancer (early / mid game)

**The list of just strong bots:**
- Arc Warden
- Dazzle
- Death Prophet (spirit siphon)
- Disruptor
- Drow Ranger
- Earth Spirit
- Enchantress
- Juggernaut (early game)
- Lion
- Lycan
- Necrophos
- Night Stalker
- Oracle
- Pudge
- Skywrath Mage
- Slardar
- Ursa

**The list of Medium difficulty:**
- Antient Apparition
- Abaddon (ultimate)
- Brewmaster
- Crystal Maiden
- Earth Shaker
- Ember Spirit
- Keeper of the Light
- Lich
- Nyx Assassin
- Omniknight
- Outworld Destroyer
- Riki
- Terror Blade
- Treant Protector
- Witch Doctor

Something is really wrong with these guys (I've seen them playing much better):
- Alchemist
- Antimage
- Bane
- Chaos Knight
- Enigma
- Faceless Void
- Lifestealer
- Spectre
- Spirit Breaker
- Tidehunter
- Tiny (!!!  I remember a Tiny with Butterfly and BKB on my highground 5 years ago)
- Wraith King
- Zeus

**Comment by keanesnow98** [comment]
I think Bane is not bad?
And I think wraith king's bot script must be updated.
Because it's really a strong hero when human plays it.

BTW, I have downloaded source code directly from this repo, how can I test in my solo bot mode with this?

**Comment by adamqqqplay** [comment]
> BTW, I have downloaded source code directly from this repo, how can I test in my solo bot mode with this?

You can check out this page to start a local game: https://github.com/adamqqqplay/dota2ai#play-offline

**Comment by keanesnow98** [comment]
> You can check out this page to start a local game: https://github.com/adamqqqplay/dota2ai#play-offline

I have already checked that guide.
So, this script is only available for ranked matching?
I have downloaded this source code from github, copied to .../scripts/vscripts and then extract the folder and renamed as `bots`.
But in solo bot mode, I can only see `Default Bot` script listed...

**Comment by adamqqqplay** [comment]
> I have already checked that guide. So, this script is only available for ranked matching?

No, although this script is called "Ranked Matchmaking AI", it cannot be used for ranked matchmaking. It's just that at the beginning, we hope that this script can achieve the feeling of ranked matchmaking.

> I have downloaded this source code from github, copied to .../scripts/vscripts and then extract the folder and renamed as `bots`. But in solo bot mode, I can only see `Default Bot` script listed...

In this case, the `Default Bot` is replaced by our script. If it still doesn't work, you can create a game lobby first, and then select the local dev script in the bot list, remember to choose LOCAL HOST in the server location.

If you see the bot's name ending in `.RMMAI`, then the script has been successfully activated.

**Comment by keanesnow98** [comment]
Thanks for your guidance.
Yes, I can see all bots' names are changed with ending **.RMMAI** which I didn't use to see these names before!
So, something changed, but I am still wondering why bots are still choosing only few heroes?
For example, they still don't pick heroes like anti-mage, Invoker or Doom Brigner, so on. Can you help me with this?

**Comment by keanesnow98** [comment]
Thanks again, @adamqqqplay! I will try that way.
